### ISSUE TITLE
ExceptionInInitializerError on 1.17

### ISSUE THREAD

**Issue by SlimeDog** [Photon-GitHub/AACAdditionPro#440]
Spigot 1.17-3160
ProtocolLib 4.7.0-b521
AACAdditionPro 7.0.0
default config.yml

On server start-up
```
[19:27:58] [Server thread/ERROR]: Error occurred while enabling AACAdditionPro v7.0.0 (Is it up to date?)
java.lang.ExceptionInInitializerError: null
        at de.photon.aacadditionpro.modules.ModuleManager.<clinit>(ModuleManager.java:88) ~[?:?]
        at de.photon.aacadditionpro.AACAdditionPro.onEnable(AACAdditionPro.java:120) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:495) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:409) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:608) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at net.minecraft.server.dedicated.DedicatedServer.init(DedicatedServer.java:264) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:987) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:307) ~[spigot-1.17-3160.jar:3160-Spigot-f773da8-295ab08]
        at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.lang.IllegalArgumentException: Cannot find associated packet class: TRANSACTION[PLAY, SERVER, 246, classNames: [net.minecraft.network.protocol.game.PacketPlayOutTransaction, net.minecraft.network.protocol.game.ClientboundTransactionPacket, net.minecraft.network.play.server.SPacketConfirmTransaction] (unregistered)]
        at com.comphenix.protocol.injector.StructureCache.newPacket(StructureCache.java:99) ~[?:?]
        at com.comphenix.protocol.events.PacketContainer.<init>(PacketContainer.java:133) ~[?:?]
        at de.photon.aacadditionpro.util.packetwrappers.sentbyserver.WrapperPlayServerTransaction.<init>(WrapperPlayServerTransaction.java:13) ~[?:?]
        at de.photon.aacadditionpro.modules.checks.pingspoof.Pingspoof.<clinit>(Pingspoof.java:39) ~[?:?]
        ... 12 more
```

**Comment by KolakZ** [comment]
The plugin got recoded in `7.0.0`, delete your old config.

**Comment by SlimeDog** [comment]
I did. I have the default 7.0.0 config.yml

**Comment by Janmm14** [comment]
Seems like Photon did an oopsie with pingspoof support for 1.17

**Comment by Photon-GitHub** [comment]
Pingspoof should autodisable on 1.17… oh well.

**Comment by SlimeDog** [comment]
Confirmed fixed on Spigot 1.17-3160 and Paper 1.17-43. Thanks.
